actions/reconfigure.go

```diff
@@ -393,20 +392,22 @@ backend %s{{$.ServiceName}}-be{{.Port}}
 	if len(sr.Users) > 0 {
 		tmpl += `
     acl {{$.ServiceName}}UsersAcl http_auth({{$.ServiceName}}Users)
-    http-request auth realm {{$.ServiceName}}Realm if !{{$.ServiceName}}UsersAcl`
-	} else if len(os.Getenv("USERS")) > 0 {
+    http-request auth realm {{$.ServiceName}}Realm if !{{$.ServiceName}}UsersAcl
```

Comment:
Would it be possible to remove the commits from the previous PR? It would make the review much easier. Do it only if it is not too much fuss, otherwise, they can stay :)